app/Services/VoucherEvaluator/Valuation.php

```diff
@@ -133,28 +133,39 @@ public function getEligibility()
     /**
      * Grabs and flattens a specified attribute from Valuations.
      * @param $attribute
-     * @param bool $onlyEligible
+     * @param bool $onlyEligibility
      * @return mixed|null
      */
-    public function flat($attribute, bool $onlyEligible = false)
+    public function flat($attribute, bool $onlyEligibility = null)
     {
-        // Check this attribute
-        if (property_exists($this, $attribute) &&
-            // If this valuation isn't eligible
-            $this->getEligibility() === $onlyEligible
+
```

Comment:
Whitespace. don't know if you want to leave an empty line betwee the opening curly brace and the first line of code/comment.

Reply:
True, will remove